v2/apiserver/schemas/project-role-assignment.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "github.com/brigadecore/brigade/v2/project-role-assignment.schema.json",
+
+	"definitions": {
+
+    "apiVersion": {
```

Comment:
nit: for `apiVersion`, we can reference the canonical def in common.json.  So, we can remove this from definitions and set up the reference on line 27 (currently) below.

Reply:
Good catch. I wrote this before your recent (and excellent) work DRYing up the schemas. When I rebased, I forgot to take advantage of the common type definitions you established. I will fix accordingly.

Reply:
Fixed